files/zh-cn/web/html/element/heading_elements/index.html

```diff
@@ -177,9 +177,9 @@ <h4 id="嵌套">嵌套</h4>
  <li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-headings.html">Understanding Success Criterion 2.4.10 | W3C Understanding WCAG 2.0</a></li>
 </ul>
 
-<h3 id="Labeling_section_content">Labeling section content</h3>
+<h3 id="标注章节内容">标注章节内容</h3>
 
-<p>Another common navigation technique for users of screen reading software is to generate a list of <a href="/en-US/docs/Web/HTML/Element#Content_sectioning">sectioning content</a> and use it to determine the page's layout.</p>
+<p>屏幕阅读软件用户的另一种常用导航技术是生成 <a href="https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element#内容分区">内容分区</a>的列表，并使用其确定页面布局</p>
```

Comment:
just remove the `domain`:

```suggestion
<p>屏幕阅读软件用户的另一种常用导航技术是生成<a href="/zh-CN/docs/Web/HTML/Element#内容分区">内容分区</a>的列表，并使用其确定页面布局</p>
```